BREAKING.md

```diff
@@ -12,10 +12,18 @@ There are a few steps you can take to write a good change note and avoid needing
 
 ## 0.54 Breaking changes
 - [Removed `readAndParseFromBlobs` from `driver-utils`](#Removed-readAndParseFromBlobs-from-driver-utils)
+- [Removed `innerRequestHandler`](#Removed-innerRequestHandler)
 
 ### Removed `readAndParseFromBlobs` from `driver-utils`
 The `readAndParseFromBlobs` function from `driver-utils` was deprecated in 0.44, and has now been removed from the `driver-utils` package.
 
+### Removed `innerRequestHandler`
+`innerRequestHandler` is removed from `@fluidframework/request-handlers` package, and its usage is removed from `BaseContainerRuntimeFactory` and `ContainerRuntimeFactoryWithDefaultDataStore`.  If you are using these container runtime factories, attempting to access internal data stores via `request()` will result in 404 responses.
+
+If you rely on `request()` access to internal root data stores, you can add `rootDataStoreRequestHandler` to your list of request handlers on the runtime factory.
+
+It is not recommended to provide `request()` access to non-root data stores, but if you currently rely on this functionality you can add a custom request handler that calls `runtime.IFluidHandleContext.resolveHandle(request)` just like `innerRequestHandler` used to do.
```

Comment:
If main partners do not use it today, then I'd rather not advertise it, and instead repeat the message we have been saying all along - use a proxy. I.e. store a handle to the object you want to expose in some central place and ensure that external URI (through some root data store) will resolve such handle and continue with the rest of the query.
Jade was building something like that, and I (maybe incorrectly) called it URL minimizer component - it takes some (long internal) URI and spits out short (external) URI for it.

Reply:
With all of that, and the fact that getRootDataStore() asserts for non-root data stores, how does Bohemia work today when dealing with internal URIs that go through non-root data stores? It sounds like there is no way to do it (in direct way) other than through runtime.IFluidHandleContext.resolveHandle?

Reply:
> With all of that, and the fact that getRootDataStore() asserts for non-root data stores, how does Bohemia work today when dealing with internal URIs that go through non-root data stores? It sounds like there is no way to do it (in direct way) other than through runtime.IFluidHandleContext.resolveHandle?
Glancing at the code, I believe they do runtime.resolveHandle() directly, which calls into this.getDataStore(), so that's how they get around that.

That's right, they don't use these handlers.

I think we must provide clear guidance to any partner who was using these already, such that if they start seeing errors they know what to do.  I think this text strikes an OK balance of recommending against the practice but informing the customer what steps they can take to unbreak themselves.